google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/impl/AppEngineCommunityWebIntegration.java

```diff
@@ -82,18 +81,15 @@ public void setupJpaSupport(@NotNull Module module, @NotNull VirtualFile persist
   }
 
   @Override
-  public void setupRunConfiguration(@NotNull AppEngineSdk sdk,
-      @Nullable Artifact artifact,
-      @NotNull Project project) {
+  public void setupRunConfiguration(@Nullable Artifact artifact, @NotNull Project project) {
```

Comment:
So the way it works is that plugin.xml in our main cloud-tools module defines a WebIntegration inteface. The same config file then injects a Community implementation for that module.

Similarly, the ultimate module injects an overriding Ultimate implementation. This module is only loaded when running UE.

The CE version is basically just a noop implementation.